### PR TITLE
kor/used label check included into related funcs to ignore them

### PR DESCRIPTION
**Pull request by ozgurcd**
kor/used label check included into related funcs to ignore them

### CODE REVIEW

File: pkg/kor/deployments.go [hunk 1]
```diff
@@ -20,6 +20,13 @@ func getDeploymentsWithoutReplicas(kubeClient *kubernetes.Clientset, namespace s
 	var deploymentsWithoutReplicas []string
 
 	for _, deployment := range deploymentsList.Items {
+		// Skip deployments that are already marked as used with the label "kor/used"="true"
```
Comment (yonahd) [inline pkg/kor/deployments.go:23]: Can you clean the comments?

**Review by yonahd — COMMENTED** [review]
Looks good!
One minor comment.

Also If you can add something to the Readme(Not critical, I can do it later)

File: pkg/kor/deployments.go (hunk 1 above)
Comment (ozgurcd) [reply pkg/kor/deployments.go:23]: Sure.

**Review by yonahd — APPROVED** [review]
LGTM!
Thanks for the contribution

**Pull request merged**